scripts/src/models/orders.ts

```diff
@@ -101,7 +101,7 @@ export class Order extends CreationDateModel {
 	public type!: OfferType;
 
 	@Column("simple-json", { name: "blockchain_data", nullable: true })
-	public blockchainData?: BlockchainData;
+	public blockchainData!: BlockchainData;
```

Comment:
why?
it can be nullable based on the column meta

Reply:
what? now we always set the blockchain data on both marketplace and external orders

Reply:
same def as Offer table